string_challenges.py

```diff
@@ -1,28 +1,30 @@
 # Вывести последнюю букву в слове
 word = 'Архангельск'
-# ???
+print(word[-1])
 
 
 # Вывести количество букв "а" в слове
 word = 'Архангельск'
-# ???
+print(len([l for l in word if l == 'а']))
+print(len([l for l in word.lower() if l == 'а']))  # если без учета регистра
```

Comment:
Хорошо, но строковые методы будут быстрее (и кмк нагляднее) чем list comprehention. 
```suggestion
print(word.lower().count('а'))
```